.gitignore

```diff
@@ -36,4 +36,5 @@ jspm_packages
 # Optional REPL history
 .node_repl_history
 
-/dist
+# dist
+dist
```

Comment:
I don't think this was a meaningful change, either remove it or remove the useless comment.